irods_testing_environment/archive.py

```diff
@@ -212,3 +212,15 @@ def collect_files_from_containers(docker_client,
 
         for p in paths_to_copy_from_containers:
             copy_from_container(source_container, p, od)
+
+
+def put_string_to_file(container, target_file, string):
+    """Echo `string` into `target_file` in `container`.
+
+    Arguments:
+    container -- docker.Container where the target_file is hosted
+    target_file -- the path inside the container with the JSON contents to modify
+    string -- contents to echo into the target file
+    """
+    if execute.execute_command(container, f'bash -c \'echo "{string}" > {target_file}\'') != 0:
```

Comment:
Do we always want the target file to be overwritten?

Did you use _put_ in _put_string_to_file_ because of iRODS PUT/object semantics lol?

I was sitting here wondering why you chose _put_ rather than _write_ or some other word lol.

Reply:
I chose "put" because the function I stole this logic from is called `put_json_to_file`. I am happy to rename these to whatever seems more appropriate.

Reply:
No need to change. I was just wondering if iRODS influenced the naming in some way.

Proceed.

Reply:
Perhaps unconsciously :)